tests/unit/test_parsers.py

```diff
@@ -1622,6 +1622,103 @@ def test_can_parse_route53_with_missing_message(self):
         self.assertEqual(error['Message'], '')
 
 
+class TestS3ExpiresResponses(unittest.TestCase):
```

Comment:
Ideally we don't want to be adding net-new tests as unittest TestCases. We can use [pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) for what you're doing in the setup section and [`@pytest.mark.parametrize`](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html) to break the timestamps into individual tests.

The reason this is valuable is it allows us to better parallelize tests and it gives us more granular failures. When we test a bunch of use cases in a single test, it becomes harder to determine what is actually failing.